src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java

```diff
@@ -5,18 +5,40 @@
 @ToString
 public class NPCRequestPacket extends DataPacket {
 
+    public static final int REQUEST_SET_ACTIONS = 0;
+    public static final int REQUEST_EXECUTE_ACTION = 1;
+    public static final int REQUEST_EXECUTE_CLOSING_COMMANDS = 2;
+    public static final int REQUEST_SET_NAME = 3;
+    public static final int REQUEST_SET_SKIN = 4;
+    public static final int REQUEST_SET_INTERACTION_TEXT = 5;
```

Comment:
Why not to use an enum here?

Reply:
Oh, I was looking at some of the other code and it was done similar to this. I can convert this to an enum however